docs/source/contributing/dev_env.rst

```diff
@@ -2,20 +2,49 @@
 Setting up the development environment
 ======================================
 
+Before building the code, please ensure the dependencies are installed:
+
+- GTK: >=3.18
+- VTE: 2.91
+- ``libkeybinder3``
+
+See full list of dependencies on the :ref:`Dependencies` page.
+
+The following section describes how to install these dependencies on some systems, please be
+aware it might not be applicable to yours directly.
+
 Install System dependencies
----------------------------
+===========================
 
 Ubuntu
-~~~~~~
+------
 
 Execute the following command to bootstrap all needed system dependencies:
 
 .. code-block:: bash
 
     $ ./scripts/bootstrap-dev-debian.sh
 
-Setup development env
----------------------
+Fedora
+------
+
+Execute:
+
+.. code-block:: bash
+
+    $ ./scripts/bootstrap-dev-fedora.sh
+
+Arch Linux
+----------
+
+Execute:
+
+.. code-block:: bash
+
+    $ ./scripts/bootstrap-dev-arch.sh
```

Comment:
Did I miss something or does this file `./scripts/bootstrap-dev-fedora.sh` not exist?

Reply:
yes it is on my other branch at home I'll upload it tonight :)

Reply:
Didn't saw you comment. Forget my second ;)

Reply:
perfect I'll use it!

Reply:
:laughing: Ok